docker-compose.yml

```diff
@@ -4,6 +4,8 @@ services:
   db:
     image: postgres:12
     env_file: .env
+  redis:
```

Comment:
why you choose Redis ? can we use something different ?

Reply:
Yupp, there are more brokers but for example configuration of RabbitMQ seemed more difficult so I chose easier one as I don't remember u having any tips for us in that matter. Was it a bad choice?